chore(docs) more detail in ipv4 data type & add operators (#1865)
questdb/questdb.io: docs/reference/sql/datatypes.md

## IPv4

QuestDB supports the IPv4 data type. 

The data type adds validity checks and type-specific functions.

They are - as one would imagine - very useful when dealing with IP addresses.

> Within a column, all addresses are stored in full 32-bit dotted decimal format as `int`. 

Reviewer: ```suggestion
Within a column, all addresses are stored as a 32-bit `int`. 
```

> When passed as arguments they behave as `string`, and may include a netmask or a subnet with a netmask. 

Reviewer: ```suggestion
When passed as an `IPv4` argument, they may not include a netmask. When passed as a `string` argument (in functions where this is explicitly allowed), they may include a netmask or a subnet with a netmask. 
```

> Or neither.

Reviewer: ```suggestion
```

> For example, an IPv4 address stored on disk as `int`: 

Reviewer: ```suggestion
`IPv4` arguments may take the following form: 
```

* ip address without a netmask: `23.62.200.1`

> And an IPv4 address passed as a `string` argument:

Reviewer: ```suggestion
`String` arguments may take the following forms:
```
